test/src/test/java/org/corfudb/integration/cluster/Harness/Node.java

```diff
@@ -0,0 +1,37 @@
+package org.corfudb.integration.cluster.Harness;
+
+import lombok.Getter;
+
+/**
+ * An abstraction for a corfu server/node.
+ * <p>
+ * <p>Created by maithem on 7/18/18.
+ */
+public class Node {
+
+    @Getter
+    final String address;
+
+    @Getter
+    final String clusterAddress;
+
+    @Getter
+    final String logPath;
```

Comment:
The only way to run in memory mode would be to pass null as the logPath, should we instead have an explicit variable to specify if running in memory or persistent mode? or at least add a comment on the use of this...